kagenti/installer/app/resources/auth-secret.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: auth-secret
+  namespace: kagenti
+spec:
+  template:
+    spec:
+      serviceAccountName: auth-secret-sa
+      containers:
+      - name: client
+        image: ghcr.io/kagenti/auth-secret:latest
+        imagePullPolicy: Always
+        env:
+        - name: KEYCLOAK_URL
+          valueFrom:
+            configMapKeyRef:
+              name: environments
+              key: KEYCLOAK_URL
+        - name: KEYCLOAK_REALM
+          valueFrom:
+            configMapKeyRef:
+              name: environments
+              key: KEYCLOAK_REALM
+        - name: KEYCLOAK_ADMIN_USERNAME
+          valueFrom:
+            configMapKeyRef:
+              name: environments
+              key: KEYCLOAK_ADMIN_USERNAME
+        - name: KEYCLOAK_ADMIN_PASSWORD
+          valueFrom:
+            configMapKeyRef:
+              name: environments
+              key: KEYCLOAK_ADMIN_PASSWORD
```

Comment:
The Keycloak admin password should be in a Secret, not a ConfigMap.  Probably the admin username as well.

Reply:
I agree that the password should not be in the environments configmap. I'm not sure where to put it because this is statically defined.

I guess the `KEYCLOAK_ADMIN_USERNAME` and `KEYCLOAK_ADMIN_PASSWORD` should be passed in as options or environment variables for the installer? @pdettori

Reply:
People are often OK with the initial install having a hard-coded password.  The issue is that in a cluster with security we want it to be possible to set it up so that only the admin can `kubectl get` the password.  The normal way to do that is by letting only the admin view Secrets.

Reply:
We could use the `installer/app/.env` to setup sensitive keys - that is already done for other things. We could even set default user/passwords (that user could change before running the installer) in `installer/app/.env_template`